iothub/service/src/RetryPolicies/IotHubServiceRetryHandler.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using System.Threading;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Provides the base implementation of the retry mechanism for unreliable actions and transient conditions.
+    /// </summary>
+    /// <remarks>
+    /// Jitter can be under 1 second, plus or minus.
```

Comment:
What is this jitter you speak of? I don't see that in this class.

Reply:
I realize that was in the class you copied. It is nonsense, so you can remove it from both.

Reply:
Ah, this was blindly copied from "RetryHanlder.cs" in Hub device. I will remove for both places then.